extension/manifest.json

```diff
@@ -10,7 +10,8 @@
 	},
 	"permissions": [
 		"alarms",
-		"activeTab"
+		"activeTab",
+		"tabs"
```

Comment:
Without 'tabs' in the manifest, it works when a notifications tab already exists, but when there is none, it seems to overwrite whatever tab I have open. To me that indicates that I should leave out the check for 'undefined'? I'll try that.